data.ts

```diff
@@ -44,6 +45,10 @@ export async function populate(path: string, opts: ClientOptions) {
   const data = await getData(path)
   if (data) {
     const client = new Client(opts)
-    await client.bulk({ body: data })
+    const batch_size = 10
+    const batches = _.chunk(data, batch_size)
+    batches.forEach(async (batch: object[]) => {
```

Comment:
Is explicit typing necessary here?

```suggestion
    batches.forEach(async (batch) => {
```

Reply:
Not having any type seems to give me an error. We could make it _batch: any_ if that's a concern. I don't know if the code would even work with any other type though. client.bulk requires an array of objects.

Reply:
Please take a look at ronitagarwala01/architect-plugin-search#1.

Reply:
Ok, merged.